docs/ui.md

```diff
@@ -60,17 +60,20 @@ If mutiple players in a group are being streamed to then this is shown by a numb
 
 There are four ways to group players
 
-1.    **MANUAL SYNC.** As shown in the [Player List](#player-list) section players can be linked via the chain icon in the player list. Only like player types that support synced playback are allowed to be joined. These players will then play in sync. It should be noted that when players are grouped in this way only the main player holds the queue. For example, player A has an existing queue and players B and C are joined to player A. If player A is turned off, players B and C will also stop. This type of group is created when the HA action [media_player.join](https://www.home-assistant.io/integrations/media_player/#action-media_playerjoin) is used.
-2.    **SYNC GROUP.** Create a group player via the settings. The native player types shown (i.e. not the Universal Group option) support perfect sync (this is a requirement to be shown here). In addition to playing in sync the group will also keep playing regardless of which player(s) in the group are powered off. So the group player holds the queue rather than any individual player. 
+1.    **MANUAL SYNC.** As shown in the [Player List](#player-list) section players can be linked via the chain icon in the player list. Only like player types that support synced playback are allowed to be joined. These players will then play in sync. It should be noted that when players are grouped in this way only the main player holds the queue. For example, player A has an existing queue and players B and C are joined to player A. If player A is turned off the queue will be lost and players B and C will stop. This type of group is created when the HA action [media_player.join](https://www.home-assistant.io/integrations/media_player/#action-media_playerjoin) is used.
```

Comment:
the queue will not be lost when you turn off a player

Reply:
fixed